tests/dataloader/test_packed_dataset.py

```diff
@@ -26,8 +25,7 @@ def test_packed_megatron_dataset_loading(dummy_packed_data_path, block_size, exp
 def test_packed_continuous_dataset_loading(dummy_packed_data_path, block_size, expected_length, expected_output):
     ds = PackedMemMapDatasetContinuous(dummy_packed_data_path, block_size, sample_key="input_ids")
     assert len(ds) == expected_length
-    dl = LLMDataLoader(dataloader_tag="unittest", dataset=ds)
-    retrieved_input_ids = [list(x["input_ids"]) for x in dl]
+    retrieved_input_ids = [list(batch["input_ids"]) for batch in ds]
```

Comment:
This change does not make sense to me. Why does `PackedMemMapDatasetContinuous` provide a `batch` in the for loop?

Reply:
I guess you are right, it would be rather `packed_samples`. Is the other change clear (the removal of the data_loader?

Reply:
yes the other changes are clear to me. I assume the intention behind them was to have the testing more granular? 
`packed_samples` would indeed be a better naming.